WalletWasabi.Fluent/Styles/TextBox.axaml

```diff
@@ -162,8 +162,18 @@
     </Setter>
   </Style>
 
-  <Style Selector="TextBlock.monoSpaced">
-    <Setter Property="FontFamily" Value="{StaticResource MonospacedFont}" />
-    <Setter Property="FontSize" Value="15" />
+  <Style Selector="TextBox.readOnly">
+    <Setter Property="IsReadOnly" Value="True" />
+    <Setter Property="Background" Value="Transparent" />
+    <Setter Property="BorderThickness" Value="0" />
+    <Setter Property="TextWrapping" Value="Wrap" />
+  </Style>
+
+  <Style Selector="TextBox.readOnly:pointerover /template/ Border#PART_BorderElement">
+    <Setter Property="Background" Value="Transparent" />
+  </Style>
+
+  <Style Selector="TextBox.readOnly /template/ TextPresenter">
+    <Setter Property="TextBlock.FontFamily" Value="{StaticResource MonospacedFont}" />
```

Comment:
I don't know why did you delete the `monoSpaced` style. This definitely should be a separate class, it is used in a few other places too. Even if it was bugged.